module/setup.go

```diff
@@ -6,20 +6,58 @@ import (
 
 	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
 	"github.com/cosmos/ibc-go/v8/modules/core/exported"
-	"github.com/datachainlab/ibc-parlia-relay/module/constant"
 	"github.com/hyperledger-labs/yui-relayer/core"
 	"github.com/hyperledger-labs/yui-relayer/log"
 )
 
+const skip = 100
```

Comment:
It would be helpful if you could add some explanation about this.

Reply:
I added the comment by https://github.com/datachainlab/ibc-parlia-relay/pull/57/commits/49dcb36c109c350e995a497afb50127e684202c6